actions/setup-elementor-env/action.yml

```diff
@@ -9,6 +9,9 @@ inputs:
     templates:
         required: false
         description: Paths to the Elementor templates directory, separated by new lines (e.g. `./path/to/elementor-templates \n ./another/path/to/elementor-templates`)
+    media:
```

Comment:
@yotamselementor 
You can add some media dir in the `stubs` directory and test in `.github/workflows/test-actions` just add some checks in the main job.